client/app/billing/autoRenew/autorenew.routing.js

```diff
@@ -126,7 +126,10 @@ export default /* @ngInject */ ($stateProvider) => {
       sort: /* @ngInject */ $transition$ => $transition$.params().sort,
 
       updateServices: /* @ngInject */ $state => ({ serviceId }) => $state.go('app.account.billing.autorenew.update', { serviceId }),
-      updateExchangeBilling: /* @ngInject */ $state => ({ serviceId }) => $state.go('app.account.billing.autorenew.exchange', { serviceId }),
+      updateExchangeBilling: /* @ngInject */ $state => ({ serviceId }) => {
+        const [organization, exchangeName] = serviceId.split('/service/');
```

Comment:
`organization === exchangeName` so this trick might not be needed

Reply:
They are the same for now, but is ready if it should changes ;)

Reply:
It's been the same for the last decade though 😄